### DISCUSSION TITLE
Hello. Could you tell me in which file I can check if LLC belongs to the virtual index and virtual tag structure？

### DISCUSSION THREAD

**Discussion by szuhzw** [CMU-SAFARI/Hermes#37] [Q&A] [answered]
Hello. Could you tell me in which file I can check if LLC belongs to the virtual index and virtual tag structure？  @rahulbera

**Comment by szuhzw** [comment]
Hello. Would the LLC in the processor configuration in the paper be shared among multiple cores? @rahulbera

**Comment by rahulbera** [comment, marked as answer]
Yes, the LLCis shared among all the cores.

**Reply by szuhzw** [reply]
> Yes, the LLCis shared among all the cores.
By the way, does LLC use physically-indexed and physically-tagged? @rahulbera
